src/main/java/com/neighborhoodAssurance/NeighborController.java

```diff
@@ -7,9 +7,7 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
-public class NAController {
-	
-	//private ServiceStub
+public class NeighborController {

```

Comment:
I changed this to NeighborContorller because when I first saw it I thought it was NA for North America so I felt that this would clear up on naming confusion.

Reply:
Good point... Abbreviations can mean different things to different people; best to spell it out.